diff --git a/variants/portduino/platformio.ini b/variants/portduino/platformio.ini
index d37c6be2..b874c576 100644
--- a/variants/portduino/platformio.ini
+++ b/variants/portduino/platformio.ini
@@ -1,6 +1,6 @@
 [env:native]
 extends = portduino_base
-build_flags = ${portduino_base.build_flags} -O0 -I variants/portduino
+build_flags = ${portduino_base.build_flags} -O0 -I variants/portduino -lgpiod
 board = cross_platform
 lib_deps = ${portduino_base.lib_deps}
 build_src_filter = ${portduino_base.build_src_filter}
\ No newline at end of file
